apps/src/templates/sectionsRefresh/QuickAssignTable.jsx

```diff
@@ -1,8 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import moduleStyles from './sections-refresh.module.scss';
 
 export default function QuickAssignTable({marketingAudience, courseOfferings}) {
-  return <div>{JSON.stringify(courseOfferings[marketingAudience])}</div>;
+  const TABLE_COUNT = Object.keys(courseOfferings[marketingAudience]).length;
+
+  const renderTable = headerIndex => {
+    return (
+      <table className={moduleStyles.table}>
+        <thead>
+          <tr className={moduleStyles.headerRow}>
+            <td className={moduleStyles.headerCell}>
+              <div>
+                {Object.keys(courseOfferings[marketingAudience])[headerIndex]}
+              </div>
+            </td>
+          </tr>
+        </thead>
+        <tbody>
+          {JSON.stringify(
+            Object.values(courseOfferings[marketingAudience])[headerIndex]
```

Comment:
This is temporary: I am making sure I can pull in alllll the info but recognize there's much more organizing to be done here.